addons/account_payment/wizards/payment_link_wizard_views.xml

```diff
@@ -11,4 +11,30 @@
         <field name="binding_view_types">form</field>
     </record>
 
+    <record id="payment_link_wizard_view_form_inherit_account_payment" model="ir.ui.view">
+        <field name="name">payment.link.wizard.form.inherit</field>
+        <field name="model">payment.link.wizard</field>
+        <field name="inherit_id" ref="payment.payment_link_wizard_view_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='amount_max']" position="attributes">
+                <attribute name="string">Amount Due</attribute>
+                <attribute name="readonly">True</attribute>
```

Comment:
This will change the view for all record types, not only account.move, as soon as `account.payment` is installed.

Reply:
Right. I added a computed field that takes the value from `account_max` and only show it now in the view when the `res_model == 'account.move'`